src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -62,19 +60,17 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
     const daysUntilNextMonday = (7 - today + 1) % 7;
     startDate.setDate(startDate.getDate() + daysUntilNextMonday);
   }
-  const { reset, handleSubmit, control } = useForm<FormInput>({
-    resolver: yupResolver(schema),
-    defaultValues: {
-      confirmDetails: false,
-      startDate
-    }
-  });
 
   const defaultValues: FormInput = {
-    startDate: new Date(),
+    startDate: startDate,
     confirmDetails: false
   };
 
```

Comment:
You can probably get rid of the defaultValues on line 64 and on line 71, do defaultValues: { confirmDetails: false, startDate: startDate } to simplify the code.